src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al

```diff
@@ -274,6 +295,7 @@ page 149042 "AIT CommandLine Card"
         CannotFindAITSuiteErr: Label 'The specified Test Suite with code %1 cannot be found.', Comment = '%1 = Test Suite id.';
         AITCode: Code[100];
         LineNoFilter: Integer;
+        LanguageTag: Text[80];
```

Comment:
Shouldn't this one be set form the OnAfterGetRecord trigger or OnAfterGetCurrentRecord?

Reply:
This is on the command line card, to set the language to run. Updated the name and tooltip to more clearly reflect.